version: "3.9"
services:
  jupyter:
    depends_on:
      - pg
    image: mpl_jupyter
    ports:
      - "62000:8888"
    volumes:
      - ./:/home/jovyan/app:Z
    command: wait-for-it.sh pg:5432 -- python -u bin/02_registra.py
    # restart: on-failure:10
   
  pg:
    image: postgres:latest
    ports: 
      - "5432:5432"
    volumes:
      - pg_temperatura:/var/lib/postgresql/data:Z
    environment:
      - POSTGRES_HOST_AUTH_METHOD=trust

volumes:
  pg_temperatura:
    external: true
    name: pg_temperatura
